lib/rules/function-calc-no-unspaced-operator/index.cjs

```diff
@@ -53,6 +54,8 @@ const rule = (primary, _secondaryOptions, context) => {
 
 			if (!OPERATOR_REGEX.test(value)) return;
 
+			if (!CALC_FUNC_REGEX.test(value)) return;
```

Comment:
Does this rule also check other match functions?

```
width: min((10px+20px), 5px);
```

Reply:
No, this rule checks only `calc()`. See also:

https://github.com/stylelint/stylelint/blob/7c0cc197f5a8040115810795a14ae275509e5e67/lib/rules/function-calc-no-unspaced-operator/index.mjs#L318